payjoin-cli/src/app/v2.rs

```diff
@@ -349,6 +382,36 @@ async fn unwrap_ohttp_keys_or_else_fetch(config: &AppConfig) -> Result<payjoin::
     }
 }
 
+struct InterruptHandler {
+    _inner: Arc<AtomicBool>,
+    tx: broadcast::Sender<()>,
+}
+
+impl std::ops::Deref for InterruptHandler {
+    type Target = broadcast::Sender<()>;
+
+    fn deref(&self) -> &Self::Target { &self.tx }
+}
+
+impl std::ops::DerefMut for InterruptHandler {
+    fn deref_mut(&mut self) -> &mut Self::Target { &mut self.tx }
+}
```

Comment:
probably fine to replace this with an impl block that calls tx.subscribe() instead